views/components/ChartList.jsx

```diff
@@ -45,54 +51,117 @@ const ChartList = () => {
       console.error(error, '*****')
     }
   }
+  const handleShareChart = (chart) => setChartToShare(chart)
+  const shareWithUsers = async (chart_id, sharedWith) => {
+    try {
+      const {
+        data: { ok },
+      } = await shareChart(chart_id, sharedWith)
+
+      if (ok === 1) {
+        const { data } = await getCharts()
+        setChartList(data)
```

Comment:
why do we need to get charts after we share a chart?

Reply:
We should hydrate the list after making an update to the chart so that it reflects the most recent changes.